plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/subscriptions-expiring-notice.tsx

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
This file looks pretty much the same with `subscriptions-expired-notice.tsx` — only class name and params differ. Probably we can merge and use one component?

```diff
13c13
< export default function SubscriptionsExpiredNotice(): JSX.Element | null {
---
> export default function SubscriptionsExpiringNotice(): JSX.Element | null {
15c15
< 	const notice = wccomSettings?.subscription_expired_notice || {};
---
> 	const notice = wccomSettings?.subscription_expiring_notice || {};
22c22
< 		const data = { notice_id: 'woo-subscription-expired-notice' };
---
> 		const data = { notice_id: 'woo-subscription-expiring-notice' };
32c32
< 			id="woo-subscription-expired-notice"
---
> 			id="woo-subscription-expiring-notice"
```

Reply:
Although we technically can, however they have different notice dismissal id parameter and they also have different Tracks events (for shown, click, dismiss).

Having them in separate components will make it easier for us if we later choose to remove one (probably if we think that the Track events for one of them are showing a bad conversion)

Reply:
I'll try to make them compact (probably we can use 1 shared component and pass some kind of type)

Reply:
yep. It seems make a sense to unify them altogether and those params can be configured with component props

Reply:
this is addressed in the newer commit